nginx/prod/nginx.conf

```diff
@@ -1,6 +1,25 @@
 server {
-	listen 80;
-	server_name localhost 127.0.0.1;
+    listen 80;
+    server_name cerberus.cadair.dev;
```

Comment:
hardcoded servername for prod?

Reply:
I replaced `nginx.conf` with a `nginx.sample.conf` so that others can edit under commit [f75853c](https://github.com/MatrixDoctors/matrix-cerberus/pull/59/commits/f75853cf9b409fba731ad13fdea71cf7e82862f5)

Is that fine?